include/vicon/udp.h

```diff
@@ -248,7 +283,7 @@ class UDPClient
     BOB_PACKED(struct RawObjectData {
         uint8_t unused;
         uint16_t itemDataSize;
-        char objectName[24];
+        CharArray<24> objectName;
```

Comment:
I still don't see why this is necessary - especially as it leads to the whole COMMA grossness. I don't see any reason not to have this as a ``char[24]`` and the map's key as a ``CharArray<24>``. You've added a ``const char *`` constructor so you won't even need to change anything else.ee